lib/opencensus/trace/samplers/probability.rb

```diff
@@ -28,12 +28,17 @@ def initialize rate, rng: nil
 
         def call opts = {}
           span_context = opts[:span_context]
-          value = if span_context
-                    (span_context.trace_id % 0x10000000000000000).to_f /
-                      0x10000000000000000
-                  else
-                    @rng.rand
-                  end
+          value =
+            if span_context
+              if span_context.sampled?
+                0.0
```

Comment:
short circuit returning true here might be easier to read

Reply:
done